docs/control-structures.rst

```diff
@@ -604,17 +599,24 @@ The error code supplied with the error data indicates the kind of panic.
 #. 0x41: If you allocate too much memory or create an array that is too large.
 #. 0x51: If you call a zero-initialized variable of internal function type.
 
-The ``require`` function either creates an error of type ``Error(string)``
-or an error without any error data and it
+The ``require`` function either creates an error without any data,
+an error of type ``Error(string)`` and it
 should be used to ensure valid conditions
 that cannot be detected until execution time.
 This includes conditions on inputs
 or return values from calls to external contracts.
 
-A ``Error(string)`` exception (or an exception without data) is generated
+.. note::
+
+    It is currently not possible to use custom errors in combination
+    with ``require``. Please use ``if (!condition) revert CustomError();`` instead.
+
+An ``Error(string)`` exception (or an exception without data) is generated
```

Comment:
I didn't checked yet the whole documentation, maybe its already described somewhere, however wouldn't it make sense to have a similar section for `Panic(uint256)`?

Reply:
There is one above this text. Starts with `A Panic exception is generated in the following situations.`